src/components/AccessibilityDocumentsList/index.tsx

```diff
@@ -146,8 +148,12 @@ const AccessibilityDocumentsList = ({
         <tbody {...getTableBodyProps()}>
           {rows.map(row => {
             prepareRow(row);
+
+            // This is used only during testing
+            const path = new URL(row.original.url).pathname;
+
             return (
-              <tr {...row.getRowProps()}>
+              <tr data-testpath={path} {...row.getRowProps()}>
```

Comment:
I don't think this is leaking anything from a security perspective. The url itself is available, and this is just parsing that. Is there any benefit to doing this parsing in the tests instead of in the app code? I have a slight aversion to code that exists just for tests, but not so much that I would value super complicated tests over comparatively simple app code.

Reply:
We can do the parsing in the test, sure. That seems a little better to me, too.